src/todo.rs

```diff
@@ -2,6 +2,16 @@ use std::fmt::{Display, Formatter, Result};
 
 pub struct Todo {
     pub message: String,
+    pub resolved: bool,
+}
+
+impl Todo {
+    pub fn resolve(&self) -> Self {
+        Self {
+            message: self.message.clone(),
+            resolved: true,
+        }
```

Comment:
Hmmm, acho que aqui ficaria mais idiomático se a atualização fosse feita in-place.

Reply:
Eu ponderei nessa hora mesmo, eu vou abstrair esse método e fazer a atualização onde é chamado, não tenho certeza se seria assim a sua sugestão. WDYM? :thinking: